typing_extensions/README.rst

```diff
@@ -0,0 +1 @@
+Please see ``README.md``.
```

Comment:
Could we make this a link? Not sure how smart GH's RST renderer is.

Reply:
Seems to work fine: https://github.com/python/typing/blob/a326f37fbc78f25b988f7ecbbdc49b5900c4884a/typing_extensions/README.rst